modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="opp-architecture-installing-policyset_{context}"]
+= Installing {product-title} by using policy sets
+
+The {ocp} installation uses two policy sets to install additional products.
+
+[NOTE]
+====
+Edit the `policyGenerator.yaml` file to remove any products that you do not want to install. You can delete the product entries or comment out the lines.
+====
+
```

Comment:
Add:
Procedure using Gitops

Follow the [Deploying policies using gitops](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/governance/governance#deploying-policies-using-gitops) procedure to install the policy sets using ACM.  Use the path to the policy set instead of the path provided in the sample: `policygenerator/policy-sets/stable/openshift-plus`.

Change: Procedure
to: Procedure using the CLI